### PR TITLE
show project name instead of repo name on navbar

### LINKED ISSUE #56: Show project name on navbar

### PR DESCRIPTION
**Pull request by yurichandra**
### Summary

Show project name on navbar

Closes #56 

### Other Information

Screen Shot 2020-10-01 at 21 59 57

### CODE REVIEW

File: app/views/layouts/exception_hunter/application.html.erb [hunk 1]
```diff
@@ -24,7 +24,7 @@
       <div class="row">
         <div class="column column-40">
           <%= link_to errors_path do %>
-            <div class="nav__title">Exception Hunter</div>
+            <div class="nav__title"><%= Rails.application.class.module_parent_name %></div>
```
Comment (brunvez) [inline app/views/layouts/exception_hunter/application.html.erb:27]: I think something like 
```suggestion
            <div class="nav__title">Exception Hunter / <%= Rails.application.class.module_parent_name %></div>
```

would look better

**Review by brunvez — COMMENTED** [review]
Hey @yurichandra thanks for the PR, could you add a screenshot of the change?

File: app/views/layouts/exception_hunter/application.html.erb (hunk 1 above)
Comment (yurichandra) [reply app/views/layouts/exception_hunter/application.html.erb:27]: Will update it and attach a screen shot

**Review by brunvez — APPROVED** [review]
Thanks!

**Pull request merged**